### DISCUSSION TITLE
Default custom module labels should be more useful

### DISCUSSION THREAD

**Discussion by k2xl** [blakeblackshear/frigate#2496] [Ideas]
I know we can load custom modules, but I think the default has some... less useful ones.

Specifically:
```
traffic light
fire hydrant
stop sign
parking meter
bench
backpack
umbrella
handbag
tie
suitcase
frisbee
skis
snowboard
sports ball
kite
baseball bat
baseball glove
skateboard
surfboard
tennis racket
bottle
wine glass
cup
fork
knife
spoon
bowl
banana
apple
sandwich
orange
broccoli
carrot
hot dog
pizza
donut
cake
chair
couch
potted plant
bed
dining table
toilet
tv
laptop
mouse
remote
keyboard
cell phone
microwave
oven
toaster
sink
refrigerator
book
clock
vase
scissors
teddy bear
hair drier
toothbrush
```

I can't think of a reason why any of those would ever be triggered in Home Assistant. A fire hydrant by design... doesn't move much :-).

There are some labels that I can imagine would be more useful by default. For example, some animals that I could see an average user wanted to be able to detect:
```
deer
squirrel
coyote
```

Also some other types of vehicles
```
ups_truck
fedex_truck
mail_truck
```
Or other types of objects/more specific types of humans
```
package
adult
child
```

**Comment by k2xl** [comment]
Note just saw https://github.com/blakeblackshear/frigate/discussions/1043 . Though I do wonder if we can still have a better default one that comes with frigate?

**Comment by peter247** [comment]
Most of the time I think you can condense all animals to animal , because a cat will be spotted and say 84% dog , stop for a second and say 70% cat , stop and say 80% dog , which is true for dogs it`s a 50/50 odds if it gets it right .

**Comment by zoltanszalontay** [comment]
Please note that the default Tensorflow Lite model used is most probably Google's Mobilenet v2 trained for Microsoft's COCO dataset, which contains these 90 labels.

More info: https://coral.ai/models/object-detection/
